### ISSUE TITLE
NVDA doesn't say anything when navigate from focused html descendants to focusable parent elements.

### ISSUE THREAD

**Issue by a11y-nvisions** [nvaccess/nvda#13587]
### Steps to reproduce:
- go to [test page](https://a11y-nvisions.github.io/quiet_bug/)
- Press a "Tab" key to navigate a div that's written 'test here!'.
- Press a "Tab" key to focus a focusable test link inside a focusable div.
- Press "Shift-Tab" to move a focus to parent div, and then check NVDA does speak an information of the focused element.

### Actual behavior:
NVDA doesn't speak information when focusing parent element again.

### Expected behavior:
NVDA should be said all even if focused on it again.

### System configuration
default

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2021.3.5

#### Windows version:
 Windows 11 Home(x64)

#### Name and version of other software in use when reproducing the issue:
Chrome 100.0.4896.75 (Official Build) (64-bit)
Firefox 99.0 (64-bit)

#### Does the issue still occur after restarting your computer?
Yes, It's always occurred.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No, only tested on 2021.3.5

#### If NVDA add-ons are disabled, is your problem still occurring?
I didn't try disabling my all add-ons. But, maybe I think my installed add-ons couldn't have any impact on this error.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
I didn't try this, too. But, maybe I think they couldn't have any impact on this error too.

**Comment by feerrenrut** [comment]
Duplicate of #6606
